File: BubbleSort/Java/src/BubbleSort.java
class BubbleSort
{
    void bubbleSort(int arr[])
    {
        int n = arr.length;
        for (int i = 0; i < n-1; i++)
            for (int j = 0; j < n-i-1; j++)
                if (arr[j] > arr[j+1])
                {
                    int temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
    }

    public static void main(String args[])
    {
        BubbleSort bubbleSort = new BubbleSort();
        int arr[] = {3, 8, 0, 1, 4, 90, 10};
        bubbleSort.bubbleSort(arr);
        System.out.println("Sorted arr");

        for (int i = 0; i < arr.length; ++i)
            System.out.print(arr[i] + " ");
        System.out.println();

    }
}

File: BubbleSort/Python/BubbleSort.py
def bubbleSort(array):
    n = len(array)
    for i in range(n - 1):
        for j in range(0, n - i - 1):
            if array[j] > array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]


array = [3, 8, 0, 1, 4, 90, 10]

bubbleSort(array)

print("Sorted array is:")
for i in range(len(array)):
    print(array[i])
